config/vsphere-csi-driver/overwrites/Chart.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: v2
 name: vsphere-csi-driver
-version: 1.0.0
+appVersion: 3.2.0
+version: 3.2.0
```

Comment:
That 1.0.0 is pretty useless if it never changes. I am matching the upstream release here.